tests/test_openai_codex_client.py

```
@@ -0,0 +1,450 @@
+            self.assertEqual(len(client.refresh_requests), 1)
+            self.assertEqual(len(client.requests), 2)
+            self.assertEqual(
+                client.requests[1][1]["Authorization"], "Bearer new-access"
+            )
+
+    def test_errors_are_sanitized(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            os.environ["CODEX_HOME"] = tmp_dir
+            _write_auth(
+                Path(tmp_dir),
+                access_token="secret-access",
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "access_token"

(S106)